app/engines/axios.js

```diff
@@ -46,18 +47,18 @@ module.exports = async (features, icao) => {
       }
     }
     if (features.chart) {
-      const lnk = $(fmtr(features.chart.selector, { icao })).attr(features.chart.attribute);
+      const lnk = $(fmtr(features.chart.selector, variables)).attr(features.chart.attribute);
       if (!lnk) {
         throw new Error('Not Found');
       }
       return `${fmtr(features.chart.baseUrl, { baseUrl: features.baseUrl })}${lnk.replaceAll(' ', '%20')}`;
```

Comment:
Can you also move `baseUrl` to the `variables` and maybe rename the `variables` to `fmtrOptions`.

Reply:
Done.

Reply:
Line 54: should pass `fmtOptions` instead of `{ baseUrl: features.baseUrl }` as the second argument to `fmtr`

Reply:
Right, I missed it.